CMakeLists.txt

```diff
@@ -101,6 +102,12 @@ foreach(line ${VERSION_INFO})
   endif ()
 endforeach()
 
+# must be global since modified FLAGS apply to other subdirectories as well, e.g., examples
+# must be after include(CompilerConfiguration) due to modifying CMAKE_Fortran_FLAGS
```

Comment:
I would still prefer this in `src/CMakeLists.txt` as a `target_compile_definitions`.
The examples currently don't seem to need it and if they do then we should rather duplicate it to give the user an example on how to do it themselves (also the example's CMakeLists.txt is an example)

Reply:
ACK. Will do.